### DISCUSSION TITLE
Hackathon 19.01.-20.01.

### DISCUSSION THREAD

**Discussion by Lica200** [SE-TINF22B2/G4-Get2Gether#78] [Statusreports]
### Allgemeines:
•	Board aufgeräumt
•	Mitarbeiterumfrage besprochen
•	Nächste Ziele
•	Teambuilding
•	Einrichtung der Entwicklungsumgebungen
### Entwicklung:
•	Startkonfiguration im Frontend
•	Angular Version
•	Vorarbeit für responsive design
•	Technischer Durchstiche beim Login mit google authentification Provider und Oberfläche
•	Nutzer- und Eventverwaltung in der Datenbank
•	Logik für Einladungslink zu Events umgesetzt
•	SwaggerUI integrierung
•	Basiswidget implementiert
### Learnings:
•	Mehr Kommunizieren, seine Probleme sofort ansprechen und auf andere zugehen
•	Rotierende PO-Rolle wird benötigt
•	Schlaf wird überbewertet (ab 4 Uhr geht’s bergauf)🚀🚀🚀
•	Schönes aufgeräumtes Board, damit jeder Bescheid weiß
•	Zusammen Programmieren ist toll
### Das lief nicht gut:
•	Terminplanung
•	Wissensverteilung
•	Ausgeruhter sein für mehr Konzentration
•	Schneller anfangen

**Comment by roland-KA** [comment]
Freut mich, dass Sie Zeit dafür gefunden und dem Projekt wieder neuen Schwung gegeben haben 😊👍
